### PR TITLE
5601 add feedback form inputs

### LINKED ISSUE #5601: Add feedback form inputs to the Help page
Add a form to the Help page for a user be able to share feedback with us.

This PR should:
- [ ] Add a new section to the help page accordian, accessible to all users, called `Contact us`
- [ ] Inside it, we should have a subheader `Feedback form` and then the two inputs for `email` and `message`
- [ ] Add a `useFeedbackForm` hook, which manages the draft state for the feedback form
- [ ] Add a `Send` button (can use [mail](https://feathericons.com/?query=mail) icon) which `console.log`s the feedback form, and shows a `success` snack that the feedback form has been saved

The radio buttons can be ignored for now, we are just adding feedback form, support request can come later:

Image

### PR DESCRIPTION
**Pull request by zachariah-at-msupply**
Fixes #5601 

# 👩🏻‍💻 What does this PR do?

- [ ] Add a new section to the help page accordian, accessible to all users, called `Contact us`
- [ ] Inside it, we should have a subheader `Feedback form` and then the two inputs for `email` and `message`
- [ ] Add a `useFeedbackForm` hook, which manages the draft state for the feedback form
- [ ] Add a `Send` button (can use [mail](https://feathericons.com/?query=mail) icon) which `console.log`s the feedback form, and shows a `success` snack that the feedback form has been saved

This is to add a form to the help page so that a user can contact us.
This is part of fulfilling the DFAT GEDSI project's objectives.

This is what the updated help page looks like (click to enlarge):

Screenshot 2024-12-03 at 11 06 14

## 💌 Any notes for the reviewer?

- Is the UI in accordance with our guidelines (especially the mail icon)?
- Is the file structure (found in `/client>pakages>host>src>Help`) in accordance with our guidelines?
- Is the styling code (specifically the way the feedback form's components are wrapped in <DetailContainer><Grid> tag, and the mail icon's file in /client/packages/common/src/ui/icons/Mail.tsx) in accordance with our guidelines?

# 🧪 Testing

1. Open the help page
2. Enter stuff in the forms
3. Click the send button
4. Do the above at different screen sizes
5. See if it is rendering and behaving as expected at all those different sizes

# 📃 Documentation

**Part of an epic**: documentation will be completed for the feature as a whole

### CODE REVIEW

File: client/packages/host/src/Help/FeedbackForm.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        labelWidth="200"
+        Input={
+          <TextArea
+            value={draft.message}
+            onChange={e => {
+              updateDraft({ message: e.target.value });
+            }}
+            InputProps={{
+              sx: {
+                backgroundColor: theme => theme.palette.background.menu,
```
Comment (lache-melvin) [inline client/packages/host/src/Help/FeedbackForm.tsx:56]: Nice use of the theme object!

File: client/packages/host/src/Help/FeedbackSection.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  const t = useTranslation();
+  return (
+    <DetailContainer>
+      <Grid
+        display="flex"
+        flex={1}
+        flexDirection="column"
+        container
+        paddingTop={2}
+        paddingBottom={1}
```
Comment (lache-melvin) [inline client/packages/host/src/Help/FeedbackSection.tsx:20]: It looks like, between the padding here, and the padding defined the the `Help` component, we've probably got a bit too much padding. However, we're going to merge this with Aimee's docs link, so the padding might actually work out great... so let's leave for now, and we can review UI clean ups at the end of the feature if needed 🚀

**Review by lache-melvin — APPROVED** [review]
Looking slick! Thanks Zachariah and team 🔥

File: client/packages/host/src/Help/FeedbackForm.tsx (hunk 1 above)
Comment (mark-prins) [inline client/packages/host/src/Help/FeedbackForm.tsx:56]: another tip, when specifying `sx` props you can use shorthand for `palette` objects. Just specify the props on palette as text and the sx prop interprets for you:

```suggestion
                backgroundColor: 'background.menu',
```

**Comment by lache-melvin** [comment]
@zachariah-at-msupply are you able to apply Mark's suggestion and merge this PR? Looks like you will need to first merge the latest develop into this branch first, and resolve conflicts 🙏

**Pull request merged**